templates/focus-ios.xliff

```diff
@@ -393,6 +458,10 @@
         <source>Learn more.</source>
         <note>Subtitle for Send Anonymous Usage Data toggle on main screen</note>
       </trans-unit>
+      <trans-unit id="Settings.rate">
+        <source>Rate Firefox Focus</source>
```

Comment:
You don't want to hard-code Firefox Focus, this should be a variable set a run-time (and set to Klar in some cases).